src/http/routes.js

```diff
@@ -201,11 +206,6 @@ RestServer.post('/server/file/save', (req, res, next) => {
     return next();
 });
 
-RestServer.del(/^\/server\/file\/f\/(.+)/, (req, res, next) => {
```

Comment:
This is a breaking change. Please revert it or confirm that no other locations in the panel hit this endpoint.